gwml2_schema.sql

```diff
@@ -0,0 +1,102 @@
+CREATE SCHEMA groundwater;
+
+CREATE TABLE groundwater.CI_Telephone (
+	id serial PRIMARY KEY,
+	voice VARCHAR(50),
+	facsimile VARCHAR(50)
+);
+
+CREATE TABLE groundwater.CI_Address (
+	id serial PRIMARY KEY,
+	deliveryPoint VARCHAR(50),
+	city VARCHAR(150),
+	administrativeArea VARCHAR(150),
+	postalCode VARCHAR(50),
+	country VARCHAR(150),
+	electronicMailAddress VARCHAR(150)
+);
+
+CREATE TABLE groundwater.CI_OnlineResource (
+	id serial PRIMARY KEY,
+	linkage VARCHAR(150),
+	protocol VARCHAR(150),
+	applicationProfile VARCHAR(250),
+	name VARCHAR(150),
+	description VARCHAR(255),
+	function groundwater.CI_OnLineFunctionTerm
```

Comment:
Mistake?

Reply:
Also be careful of using PG reserved words for field names

Reply:
it's in gwml2_vocabularies.sql file, cause you said in the tickets to separate both, though i can put both into one so you can just copy and paste